Gemfile

```
@@ -12,40 +11,42 @@ gem 'aws-sdk-iam'
 gem 'aws-sdk-route53'
 gem 'aws-sdk-s3'
 gem 'aws-sdk-sns'
-gem 'puma'
-gem 'active_model_serializers'
-gem 'gibberish'
-gem 'octokit'
 gem 'delayed_job_active_record'
-gem 'pundit'
+gem 'gibberish'
 gem 'honeybadger', require: false
 gem 'jbuilder'
 gem 'lograge'
 gem 'net-ssh'
-gem 'thor', require: false
+gem 'octokit'
+gem 'pg'
+gem 'puma'
+gem 'pundit'
+gem 'rails', '~> 5.2.4.3'
 gem 'sqlite3'
+gem 'thor', require: false
 gem 'vault'
 
 group :development, :test do
-  gem 'pry-rails'
-  gem 'pry-byebug'
   gem 'debase'
+  gem 'pry-byebug'
+  gem 'pry-rails'
   gem 'ruby-debug-ide'
   gem 'solargraph'
 end
 
 group :development do
+  gem 'rubocop'
```

Personally, I like to set the version for rubocop in the gemfile. Sometimes they update their cops and it breaks randomly.
```suggestion
  gem 'rubocop', '~> 0.86.0'
```

---

Thanks! that makes sense!